Code_Changes.js

```diff
@@ -51,6 +53,19 @@ function getChangeRequestForm() {
     return html;
 }
 
+
+// getCovidManufacturingForm : n/a -> HTML
+// return HTML display output for COVID manufacturing google form
```

Comment:
Let's use JSDoc moving forward. [Using it in GAS](https://subscription.packtpub.com/book/web_development/9781785882517/9/ch09lvl1sec80/using-jsdoc-annotations), [types](https://jsdoc.app/tags-type.html). Don't worry about updating old functions, just apply this to new ones moving forward.

```suggestion
/**
 * Provides HTML content for embedding COVID Manufacturing Google Form.
 *
 * @return {!string} HTML for embedded Google form
 */
```